source/software/r_package_management.rst

```diff
@@ -105,17 +88,18 @@ If the result is blank, or reports that conda can not be found, modify
 the \`PATH\` environment variable appropriately by adding miniconda's bin
 directory to PATH.
 
-Creating a new conda environment is straightforward::
+The next step is to create a new conda environment which can be done as follows::
+
+   $ conda create -n science -c conda-forge r-base=4.3.2 r-essentials
```

Comment:
I am somewhat divided on this one. I get that providing a version will guarantuee (or should as it's Conda) reproducability, but in one year this version might be outdated, so this would require a regular update in the documentation. What about:

conda create -n science -c conda-forge r-base=< version> r-essentials

Reply:
I propose something like the following:

The next step is to search for a desired/recent release of `r-base` package (specified below as `<version>`) and specify that version in creating a new conda environment: 

```
conda search -c conda-forge r-base  # select one of available versions for the step below
conda create -n science -c conda-forge r-base=<version> r-essentials
```

Reply:
I quite like Ehsan's suggestion.